style.css

```diff
@@ -0,0 +1,667 @@
+body {
+  margin: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: gray;
+}
+
+/* Registration page style */
+
+#registration_page {
+  background-color: #F0F8EC;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 844px;
+  width: 390px;
+}
+
+.registration_page_textbox {
+  background-color: white;
+  border: solid black 1px;
+  border-radius: 25px;
+  height: 54px;
+  width: 216px;
+  padding: 10px;
+  margin-block-start: 0em;
+}
+
+.registration_page_textbox:nth-of-type(5) {
+  margin-block-end: 2em;
+}
+
+.registration_page_form_heads {
+  color: darkslategray;
+  margin-block-end: .2em;
+}
+
+#register_button {
+  background-color: #DDEED6;
+  border: none;
+  height: 54px;
+  width: 216px;
+  border-radius: 25px;
+  font-size: 36px;
+  color: darkslategray;
+}
+
+.choose_login {
+  margin-block-start: 0em;
+  margin-block-end: 0em;
+}
+
+#registration_page {
+  background-color: #F0F8EC;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 844px;
+  width: 390px;
+}
+
+.registration_page_textbox {
+  background-color: white;
+  border: solid black 1px;
+  border-radius: 25px;
+  height: 54px;
+  width: 216px;
+  padding: 10px;
+  margin-block-start: 0em;
+}
+
+.registration_page_textbox:nth-of-type(5) {
+  margin-block-end: 2em;
+}
+
+.registration_page_form_heads {
+  color: darkslategray;
+  margin-block-end: .2em;
+}
+
+#register_button {
+  background-color: #DDEED6;
+  border: none;
+  height: 54px;
+  width: 216px;
+  border-radius: 25px;
+  font-size: 36px;
+  color: darkslategray;
+}
+
+.choose_login {
+  margin-block-start: 0em;
+  margin-block-end: 0em;
+}
+
+/* Login Page style */
+
+#login_page {
+  background-color: #F0F8EC;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 844px;
+  width: 390px;
+}
+
+.login_page_textbox {
+  background-color: white;
+  border: solid black 1px;
+  border-radius: 25px;
+  height: 54px;
+  width: 216px;
+  padding: 10px;
+  margin-block-start: 0em;
+}
+
+.login_page_textbox:nth-of-type(2) {
+  margin-block-end: 2em;
+}
+
+.login_page_form_heads {
+  color: darkslategray;
+  margin-block-end: .2em;
+}
+
+#login_button {
```

Comment:
In that vein, you're defining `background-color` a bunch of different times. It might be good to make a few different common classes to define your app colors so those stay consistent